README.md

```diff
@@ -501,6 +504,158 @@ ASP.NET Integration
   var airbrake = HttpContext.Features.Get<IAirbrakeFeature>().GetNotifier();
   ```
 
+NLog Integration
+----------------
+
+### Airbrake NLog target
+
+Airbrake NLog target is used to pass an exception from log messages to the
+Airbrake dashboard.
+
+1. Install the `Sharpbrake.NLog` package from NuGet (you can use "Package
+   Manager Console" from Visual Studio):
+
+   ```
+   PM> Install-Package Sharpbrake.NLog
+   ```
+
+2. Register the `Sharpbrake.NLog` assembly in the `<extensions />` section
+   of the [nlog.config][nlog-config] file:
+
+   ```xml
+   <extensions>
+     <add assembly="Sharpbrake.NLog"/>
+   </extensions>
+   ```
+
+3. Define the Airbrake target in the `<targets />` section and appropriate
+   routing rules in the `<rules />` section of `nlog.config`:
+
+   ```xml
+   <targets>
+     <target name="airbrake"
+             type="Airbrake"
+             projectId="113743"
+             projectKey="81bbff95d52f8856c770bb39e827f3f6"
+             environment="live"
+             ignoreEnvironments="dev"
+     />
+     <!-- other targets -->
+   </targets>
+   ```
+
+   ```xml
+   <rules>
+     <logger name="*" minlevel="Error" writeTo="airbrake" />
+     <!-- other rules -->
+   </rules>
+   ```
+
+   Note that both `projectId` and `projectKey` are required parameters. You can
+   set any configuration option supported by the Airbrake client in the declarative
+   way ([how to configure](#configuration)).
+
+   When you need to access `Notifier` programmatically (e.g. for setting up filters
+   in your code) you can get it from the `AirbrakeTarget` object:
+
+   ```csharp
+   var airbrakeTarget = (AirbrakeTarget)LogManager.Configuration.AllTargets
+       .FirstOrDefault(t => t.GetType() == typeof(AirbrakeTarget));
+   
+   airbrakeTarget?.Notifier.AddFilter(notice =>
+   {
+       // clear environment variables with "token"-related keys
+       new List<string>(notice.EnvironmentVars.Keys).ForEach(key =>
+       {
+           if (key.ToLowerInvariant().Contains("token"))
+               notice.EnvironmentVars[key] = "[removed]";
+       });
+   
+       return notice;
+   });
+   ```
+
+### Airbrake NLog target for ASP.NET
+
+Airbrake NLog target for ASP.NET differs from the regular one in the additional
+support for the HTTP context.
```

Comment:
This will sound a bit better:

> Airbrake NLog target for ASP.NET is almost identical to the standard target with the exception of support for HTTP context.